src/setting.c

```diff
@@ -693,6 +696,32 @@ static int user_style(Client *c, const char *name, DataType type, void *value, v
     return CMD_SUCCESS;
 }
 
+static int style_sheet_file(Client *c, const char *name, DataType type, void *value, void *data)
+{
+    char *path;
+    gchar *source;
+
+    path = util_get_config_dir();
+
+    vb.files[FILES_USER_STYLE] = util_get_filepath(path, value, FALSE);
```

Comment:
Note that this contains already allocated memory and has to be freed before assigning new.